apertium_apy/apy.py

```diff
@@ -178,11 +184,17 @@ def apply_config(args, parser, apy_section):
 
 
 def parse_args(cli_args=sys.argv[1:]):
+    if pkg_resources:
+        apy_path = pkg_resources.resource_filename('apertium_apy', '')
```

Comment:
I'm still a bit confused as to why we can't just use `langNamesDefaultPath` as the second arg here and get the right result. It seems like we should be able to from the docs: https://setuptools.readthedocs.io/en/latest/pkg_resources.html#resource-extraction. Are we not setting it up as a resource in the `setup.py`?

Reply:
What happens is resource_filename looks for an installed module (apertium_apy in this case) and gives its path including apertium_apy and then appends second arguement to the path. The problem here is that we need to find the path for `langNames.db` which is just outside apertium_apy. So we need to come a step back from apertium_apy and then append `langNames.db` to it

Reply:
Ah, I see what you're saying now. In that case, should using `os.path.join('..', lang_names_path)` work for the second arg?

Reply:
Yeah it would work but it exceeds the number of character travis allows in a line. So I used another variable apy_path to store the path first.

Reply:
My proposal and your code are not identical. I'm suggesting using a different argument than the empty string to the `pkg_resources` command. The line length isn't relevant.

Reply:
Works perfect. Thanks! great idea.